builder/package.ubuntu-1804

```diff
@@ -0,0 +1,48 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir -p /tmp/output/ubuntu-1804
+fi
+
+fpm \
+  -s dir \
+  -t deb \
+  -v ${BUILD_NO} \
+  -n R-${R_VERSION} \
+  --deb-priority "optional" \
+  --deb-field 'Bugs: https://github.com/rstudio/r-builds/issues' \
+  --url "http://www.r-project.org/" \
+  --description "GNU R statistical computation and graphics system" \
+  --maintainer "RStudio Inc https://github.com/rstudio/r-builds" \
+  --license "GPL-2" \
+  -p /tmp/output/ubuntu-1804/ \
+  -d zip \
```

Comment:
Sort alphabetically so it is easy to find things.

Reply:
Done